tests/operators/test_python.py

```diff
@@ -856,6 +886,25 @@ def f():
 
         self._run_as_operator(f, python_version=3, use_dill=False, requirements=['dill'])
 
+    @staticmethod
+    def _initiate_pypi_server():
+        try:
+            if not os.path.exists('/root/packages'):
+                os.makedirs('/root/packages')
+            subprocess.Popen(['pypi-server'])
+            time.sleep(1.5)
+        except Exception as e:
+            pass
+
+    @staticmethod
+    def _clean_stop_pypi_server(port: bytes = b"8080"):
```

Comment:
Too complex. running system command `killall  -9 pypi-server` does the job in one line of code.. And with the proper setup/teardown sequence, you should store the pid of the process started and try to kill it with SIGTERM rather than SIGKILL first anyway - this method should only be treated as a "safety net",